src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -144,6 +144,30 @@ export const fetchDictionaryConcepts = (
   dispatch(isFetching(false));
 };
 
+export const fetchConceptsByName = query => async (dispatch) => {
+  dispatch(isFetching(true));
+  const conceptType = localStorage.getItem('type');
+  const conceptOwner = localStorage.getItem('typeName');
+  const dictionaryId = localStorage.getItem('dictionaryId');
+  const limit = 0;
+  const page = 1;
+  const url = `${conceptType}/${conceptOwner}/collections/${dictionaryId}/concepts/?includeMappings=true&${query}&limit=${limit}&page=${page}&verbose=true`;
```

Comment:
Is this method `fetchConceptsByName` searching concepts by name only? Won't results be returned when I search by class or source?

Reply:
@collinewait thanks for pointing that out. I have tried searching by class or source and no results are returned. Ultimately, it's only a search by name that works. Lastly, if there's another endpoint you suggest I use, please add it to the comments here.